.github/workflows/notebook-controller-images-updater.yaml

```
@@ -37,6 +41,7 @@ env:
   BRANCH_NAME: ${{ inputs.branch-name }}
   GENERATE_PR: ${{ inputs.generate-pr }}
   REPO_NAME: kubeflow
+  GH_TOKEN: ${{ secrets.GH_TOKEN }} 
```

I did not try it myself, but I understand that running the workflow manually, I now need to provide the token as a build parameter?

```suggestion
  GH_TOKEN: ${{ secrets.GH_TOKEN || secrets.GITHUB_TOKEN }} 
```

Can we fallback to GITHUB_TOKEN somehow so that when running manually, I don't need to hunt for the PAT token and can let it use the default?

---

It works but it does not fire the GHA test because this `GITHUB_TOKEN` does not have elevated scopes.

>I don't need to hunt for the PAT token and can let it use the default?

In the L59 there is logic for that, the user doesn't have to deal with this